pkg/apis/addtoscheme_toolchain_v1alpha1.go

```diff
@@ -0,0 +1,10 @@
+package apis
```

Comment:
why do you need to copy this file? What about importing it from the `api` module/repo instead?